recipe/meta.yaml

```diff
@@ -22,23 +22,38 @@ requirements:
   host:
     - python
     - pip
-    - numpy
+    - setuptools
+    - wheel
+    - numpy   1.19  # [py<310]
+    - numpy   1.21  # [py==310]
+    - numpy   1.23  # [py>=311]
     - msinttypes       # [win and py<35]
```

Comment:
This can be removed since only py35 is not supported anymore: https://github.com/storpipfugl/pykdtree/blob/v1.3.6/setup.py#L104

You also need to add a skip for those versions